cmd/tpm2_autounlock/tpm2_autounlock.sh

```diff
@@ -0,0 +1,176 @@
+#! /bin/bash
```

Comment:
Can this be plain `/bin/sh`? It looks like you have a bashism (or few) in here (I see a `[[ ... ]]`), but they could probably be fixed.